pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -100,63 +105,27 @@ func Test_SubscribeLeafOnce(t *testing.T) {
 		t.FailNow()
 	}
 
-	pathResponse := responseReq.GetUpdate().Update[0].Path
```

Comment:
On line 92 above:
		err = server.Subscribe(serverFake)
this err never gets read because it's on a different thread. Can be rearranged in to 1 line as:
	go server.Subscribe(serverFake)

On line 98 above:
responseReq := <-responsesChan
Should this be in a for loop?

Reply:
on line 92, agreed, moving in the routine
on line 98 no, I'm expecting 1 reply because it's ONCE